test/methods/stream/render-comments.js

```diff
@@ -18,7 +18,7 @@ export default function renderComments () {
 
 		stream.renderComments();
 
-		const scriptTag = document.querySelector('script[src="https://ft.coral.coralproject.net/assets/js/embed.js"]');
+		const scriptTag = document.querySelector('script[^src="https://ft.coral.coralproject.net/assets/js/embed.js"]');
```

Comment:
Looks like the caret is in the wrong place in this PR (okay in the other one)

```suggestion
		const scriptTag = document.querySelector('script[src^="https://ft.coral.coralproject.net/assets/js/embed.js"]');
```